src/main/java/frc/robot/classes/Auton.java

```diff
@@ -20,12 +20,11 @@
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.InstantCommand;
 import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
-import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import edu.wpi.first.wpilibj2.command.WaitCommand;
-import frc.robot.classes.FieldConstants;
```

Comment:
This is confusing.  Not sure that it should work without this import, but it does.